src/opentime/rationalTime.cpp

```diff
@@ -193,8 +193,13 @@ RationalTime::to_timecode(
 ) const {
 
     *error_status = ErrorStatus();
+
+    double value_in_target_rate = _value;
```

Comment:
The goal of this is to convert the time to a frame count. I think the logic would be clearer if rather than using `value` and `rate` from the input `RationalTime` in this method, we instead used `RationalTime::to_frames` to get the frame count and never let the member `_rate` from the `RationalTime` instance enter this function's calculations. This clarifies the distinction between `RationalTime` `rate` (which is an arbitrary time division used to accurately express a time in seconds), and fps (which is a scheme of recording samples).

This is a bit pedantic, but it's akin to the distinction between pixels and little squares.

Reply:
As discussed, since we'll be looking more at this more in #613 we should keep it as-is t limit impact